tests/runner/src/main/java/org/mule/test/runner/api/AetherClassPathClassifier.java

```diff
@@ -357,18 +357,18 @@ public ArtifactsUrlClassification classify(final ClassPathClassifierContext cont
   }
 
   /**
-   * Gets the direct dependencies filter to be used when collecting Container dependencies.
-   * If the rootArtifact is a {@link ArtifactClassificationType#MODULE} it will include
-   * {@value org.eclipse.aether.util.artifact.JavaScopes#COMPILE} dependencies too if not just
-   * {@value org.eclipse.aether.util.artifact.JavaScopes#PROVIDED}.
+   * Gets the direct dependencies filter to be used when collecting Container dependencies. If the rootArtifact is a
+   * {@link ArtifactClassificationType#MODULE} it will include {@value org.eclipse.aether.util.artifact.JavaScopes#COMPILE}
+   * dependencies too if not just {@value org.eclipse.aether.util.artifact.JavaScopes#PROVIDED}.
    *
    * @param rootArtifactType the {@link ArtifactClassificationType} for rootArtifact
    * @return {@link Predicate} for selecting direct dependencies for the Container.
    */
   private Predicate<Dependency> getContainerDirectDependenciesFilter(ArtifactClassificationType rootArtifactType) {
     return rootArtifactType.equals(MODULE)
         ? directDep -> directDep.getScope().equals(PROVIDED) || directDep.getScope().equals(COMPILE)
-        : directDep -> directDep.getScope().equals(PROVIDED);
+        : directDep -> directDep.getScope().equals(PROVIDED)
+            || directDep.getArtifact().getClassifier().equals(MULE_PLUGIN_CLASSIFIER);
```

Comment:
@pablolagreca, I working on this code in order to remove the constraint of having to define plugins as PROVIDED for isolated tests. I was wondering if you remember why this filter has to be changed for including mule-plugins as part of the filter for the Container URL.